core/app/models/spree/reimbursement.rb

```diff
@@ -98,12 +98,16 @@ def unpaid_amount
       total - paid_amount
     end
 
-    def perform!
+    def perform!(created_by: nil)
+      unless created_by
+        created_by = Spree.user_class.where(email: 'spree@example.com').first
```

Comment:
Use `find_by` instead of `where.first`.

Reply:
Done and squashed changes into 3f33fa199e889251c517a60c4ac98be958248b4c